src/NUnitTestAdapter/NUnitTestExecutor.cs

```diff
@@ -61,17 +61,18 @@ public void RunTests(IEnumerable<string> sources, IRunContext runContext, IFrame
                 foreach (var source in sources)
                 {
                     string sourceAssembly = source;
-
                     if (!Path.IsPathRooted(sourceAssembly))
-                    {
                         sourceAssembly = Path.Combine(Environment.CurrentDirectory, sourceAssembly);
-                    }
-                    using (currentRunner = new AssemblyRunner(TestLog, sourceAssembly, tfsfilter,this))
+
+                    try // TODO: Do we need try?
```

Comment:
I left the try with a comment thinking I might end up needing to do some cleanup - turns out I forgot to clean __it__ up!

I think your idea of continuing to process other assemblies makes sense. I'll do that.